R/stat-qq.R

```diff
@@ -32,8 +32,10 @@
 #' p <- ggplot(df, aes(sample = y))
 #' p + stat_qq() + stat_qq_line()
 #'
-#' # Use fitdistr from MASS to estimate distribution params
-#' params <- as.list(MASS::fitdistr(df$y, "t")$estimate)
+#' # Use fitdistr from MASS to estimate distribution params:
+#' # params <- as.list(MASS::fitdistr(df$y, "t")$estimate)
```

Comment:
Should we put this line behind `if (requireNamespace("MASS", quietly = TRUE)) { ... }`?

Reply:
yeah, I agree